src/transformers/tokenization_utils_base.py

```diff
@@ -101,6 +101,12 @@ def __init__(
         def __getstate__(self):
             return self.__dict__
 
+        def __str__(self):
+            return self.content
+
+        def __repr__(self):
+            return self.content
```

Comment:
Nope, let's either take the default repr from the dataclass overload, or when we print the added Token we need to have: `AddedToken("▁<PRE>", rstrip=True, lstrip=True, single_word=False, normalized=False, special=True)`